tools/tpm2_createpolicy.c

```diff
@@ -112,7 +112,7 @@ static bool evaluate_populate_pcr_digests(create_policy_ctx *pctx, TPML_DIGEST *
     //total pcr indices per algorithm/ bank. Typically this is 24
     uint8_t total_indices_for_this_alg=0;
     //cumulative size total of selected indices per hashAlg
-    unsigned expected_pcr_input_file_size=0;
+    long expected_pcr_input_file_size=0;
```

Comment:
We would never expect this to go negative. The long for file_size seems to be from the ftell() call to get the file size, but it is checked for negative (negative ftell() return is error), so only values >= 0 are ever set. It looks like the interface for getting the file size should change to unsigned long.